litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx

```diff
@@ -146,7 +146,11 @@ const WorkflowSettings = forwardRef((_, ref) => {
       CRDLink,
     };
     localforage.setItem('workflow', workflowDetails);
-    if (validateWorkflowName(name)) {
+    if (name.length === 0) {
```

Comment:
!name.length can be used

Reply:
> can we remove the package lock changes? it changes with every PR without any package changes. @S-ayanide cc: @rajdas98

I needed to install the new testing library and other dependencies that were newly added, npm install will update the package-lock, which is not in our control.